pandas/io/formats/style_render.py

```diff
@@ -1105,16 +1107,28 @@ def format(
         <td .. >NA</td>
         ...
 
-        Using a ``formatter`` with LaTeX ``escape``.
+        Using a ``formatter`` with ``escape`` in 'latex' mode.
 
-        >>> df = pd.DataFrame([["123"], ["~ ^"], ["$%#"]])
+        >>> df = pd.DataFrame([["123"], ["~ ^"], ["%#"]])
```

Comment:
Its fine to change this example. Just curious as to the reason.

Reply:
> I think one change worth making, otherwise I think it is good to add it. The test demonstrates the effect nicely. Didn't check the CI tests, we make sur ethey are green before merging.

I changed uuid string to be unique, as you suggested. And also reverted the example for latex mode. I changed it by mistake, there is nothing wrong with `$` in latex mode (the online latex compiler which I used at first had some sort of bug).

CI tests look good.